homeassistant/components/sharkiq/manifest.json

```diff
@@ -1,10 +1,10 @@
 {
-  "domain": "sharkiq",
-  "name": "Shark IQ",
-  "config_flow": true,
-  "documentation": "https://www.home-assistant.io/integrations/sharkiq",
-  "requirements": ["sharkiqpy==0.1.8"],
-  "codeowners": ["@ajmarks"],
-  "iot_class": "cloud_polling",
-  "loggers": ["sharkiqpy"]
-}
+    "domain": "sharkiq",
+    "name": "Shark IQ",
+    "config_flow": true,
+    "documentation": "https://www.home-assistant.io/integrations/sharkiq",
+    "requirements": ["sharkiq==0.0.1"],
+    "codeowners": ["@ajmarks"],
+    "iot_class": "cloud_polling",
+    "loggers": ["sharkiq"]
+}
```

Comment:
Since #68877 has been merged, please add a carriage return here, or it will fail

Reply:
This should be all set now